### PR TITLE
data: weilincheng

### PR DESCRIPTION
**Pull request by weilincheng**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5491"></a>

### CODE REVIEW

File: data/weilincheng.json [hunk 1]
```diff
@@ -30,6 +30,14 @@
     }
   ],
   "milestones": [
+    {
+      "title": "Crossing the Threshold: Contributing to Open Source for the First Time",
+      "date": "March 2023",
+      "icon": "FaGithub",
+      "color": "blue",
+      "description": "Even if it's just a small enhancement, being able to contribute to the vibrant and diverse EiddeHub Community is an important milestone in my journey as a developer.",
```
Comment (khushalarora) [inline data/weilincheng.json:38]: Hi @weilincheng, there is a minor typo in the spelling of **'EddieHub'**. Please correct it :+1:

Comment (weilincheng) [reply data/weilincheng.json]: Thank you so much! I just corrected it.

**Review by khushalarora — APPROVED** [review]
Thank you :tada:

**Review by bkpecho — APPROVED** [review]
LGTM! Great job on your contribution 🎉 Have a great day ahead :nerd_face:

**Review by amandamartin-dev — APPROVED** [review]
lgtm!

**Pull request merged**